hack/jenkins/e2e.sh

```diff
@@ -93,6 +97,22 @@ GCE_DEFAULT_SKIP_TESTS=(
     "Example"
     )
 
+# Tests which cannot be run on GKE, e.g. because they require
+# master ssh access.
+GKE_REQUIRED_SKIP_TESTS=(
+    "Nodes"
+    "Etcd\sFailure"
+    "MasterCerts"
+    "Shell"
+    )
+
+GKE_REBOOT_SKIP_TESTS=(
```

Comment:
It seems that these are almost all covered by GCE_DEFAULT_SKIP_TESTS which you already skip. Can you perhaps just delete GKE_REBOOT_SKIP_TESTS ?